src/payment/strategies/bluesnapv2/bluesnapv2-payment-page-loader.ts

```diff
@@ -0,0 +1,59 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
+
+import { NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+
+import { BlueSnapV2PaymentInitializeOptions } from './bluesnapv2-payment-options';
+
+export default class BlueSnapV2PaymentPageLoader {
```

Comment:
just to be consistent with other strategies, maybe this could be called `BlueSnapV2WidgetLoader` ?

Reply:
Refactored 👍